Add OpenTelemetry Collector Builder docs (#2516)
open-telemetry/opentelemetry.io: content/en/docs/concepts/glossary.md

A mechanism for propagating name/value pairs to help establish a causal
relationship between events and services. See [baggage spec][baggage].

### **Builder**

> See [Collector Builder](#collector-builder)

Reviewer: ```suggestion
See [Collector Builder](#collector-builder).
```

Also known as the OpenTelemetry Collector. More on the Collector
[here][collector].

### **Collector Builder**

> A command-line interface tool that generates sources and binaries for
> OpenTelemetry Collector [distributions](../../collector/distributions). See more
> under the [OpenTelemetry Collector Builder (ocb)](../../collector/builder)
> documentation.

Reviewer: ```suggestion
A command-line tool that generates sources and binaries for
OpenTelemetry Collector [distributions](/docs/collector/distributions/). For more information, see [OpenTelemetry Collector Builder (ocb)](/docs/collector/builder).
```

Short form for [`OpenCensus`](#opencensus).

### **`ocb`**

> See [Collector Builder](#collector-builder)

Reviewer: ```suggestion
See [Collector Builder](#collector-builder).
```
